cartography/data/jobs/cleanup/github_repos_cleanup.json

```diff
@@ -19,6 +19,11 @@
     "iterative": true,
     "iterationsize": 100
   },
+  {
```

Comment:
Now that Dependencies are a data model, we don't need to have a manual cleanup job anymore. We can just do 

```python
    GraphJob.from_node_schema(GitHubDependencySchema(), common_job_parameters).run(
        neo4j_session,
    )
```